tests/Drupal/InfoFiles/drupal8/dependencies_array.info.yml

```diff
@@ -0,0 +1,9 @@
+core: 8.x
+name: Dependencies Array Test
+
+# These should not be here, will be added by drupal.org packaging.
```

Comment:
This and the next 3 lines can be removed for this test, we only want to test dependencies here, right?

Reply:
can you address this?